ui/src/app/utils/runtime-utils.tsx

```diff
@@ -395,6 +452,9 @@ export const useCustomRuntime = (currentWorkspaceNamespace):
                 pollAbortSignal: aborter.signal,
                 overallTimeout: 1000 * 60 // The switch to a non running status should occur quickly
               });
+            } else if (runtime.status === RuntimeStatus.Deleted && pdExists && enablePD) {
```

Comment:
the runtime deleted status check here does not make sense to me. Here we're looking to make an update.

Reply:
Here we want to update (more specifically 'Increase') an unattached PD

Reply:
the only call to `updateDisk` that I see is here, which is when the runtime is deleted.

Is this correct? Can I not resize the disk in place?

Reply:
> the only call to updateDisk that I see is here, which is when the runtime is deleted. Is this correct?

Yes, it's only used here to first increase an unattached pd and then create a new runtime with it. 

> Can I not resize the disk in place?

We resize (increase) the disk in place if it's attached to an existing runtime. But for an unattached pd, we can't use the runtime update api to resize it.